presto-main/src/test/java/com/facebook/presto/metadata/TestDiscoveryNodeManager.java

```diff
@@ -91,29 +91,34 @@ public void setup()
     public void testGetAllNodes()
     {
         DiscoveryNodeManager manager = new DiscoveryNodeManager(selector, nodeInfo, new NoOpFailureDetector(), expectedVersion, testHttpClient, internalCommunicationConfig);
```

Comment:
can you use try-with-resources for `manager`?

Reply:
No it doesn't implement `AutoCloseable`